nix/sources.nix

```diff
@@ -0,0 +1,26 @@
+# A record, from name to path, of the third-party packages
```

Comment:
you may want to grab sources.nix from iohk-ops and stick it in here as it gives you a way to point at local repos by updating NIX_PATH var for testing.